app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, origin_file, path = command.split()
+    if cmd == "mv":
+        if "/" in path:
```

Comment:
Could you think about solution which will not use "/", because in this case the script will be not multi-platform (for example, Windows users use `\`).